joomla/joomla-cms
# Adds option to combine the main jui jquery scripts

This PR adds option in the main configuration to combine the main jui jquery scripts (jquery, noconflict, migrate).

![](http://regl.io/i/0l1fx.png)

This will reduce the number of scripts getting loaded on a page.

So instead of:
```
<script src="/media/jui/js/jquery.min.js" type="text/javascript"></script>
<script src="/media/jui/js/jquery-noconflict.js" type="text/javascript"></script>
<script src="/media/jui/js/jquery-migrate.min.js" type="text/javascript"></script>
```

You will get:
```
<script src="/media/jui/js/jquery.with-noconflict.with-migrate.min.js" type="text/javascript"></script>
```

Depending on whether the noconflict and migrate are required, these are also possible:
`jquery.min.js` (no extras required)
`jquery.with-noconflict.min.js` (only noconflict required)
`jquery.with-migrate.min.js` (only migrate required)


### Testing Instructions

Switch on the option in the global configuration and check the html output of the website (both frontend and administrator side).
You should see the new combined script being loaded instead of the separate ones.


### Documentation Changes Required

This adds an option to the Global Configuration.
And therefore also adds 2 language strings.
